Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -251,8 +251,8 @@ bool PPCSymbolDB::LoadMap(const Core::CPUThreadGuard& guard, const std::string&
       continue;
 
     // Support CodeWarrior and Dolphin map
-    if (std::string_view{line}.ends_with(" section layout\n") || strcmp(temp, ".text") == 0 ||
-        strcmp(temp, ".init") == 0)
+    if (std::string_view{StripWhitespace(line)}.ends_with(" section layout") ||
+        strcmp(temp, ".text") == 0 || strcmp(temp, ".init") == 0)
```

Comment:
Do you need to construct a `string_view{}`? AFAICT, `StripWhitespace` is already returning a `string_view`, so  `StripWhitespace(line).ends_with` should work unless I'm mistaken.